File: src/main/kotlin/top/charjin/shoppingserver/mapper/OsGoodsReviewMapper.java
package top.charjin.shoppingserver.mapper;

import top.charjin.shoppingserver.entity.OsGoodsReview;

public interface OsGoodsReviewMapper {
    int deleteByPrimaryKey(Integer id);

    int insert(OsGoodsReview record);

    int insertSelective(OsGoodsReview record);

    OsGoodsReview selectByPrimaryKey(Integer id);

    int updateByPrimaryKeySelective(OsGoodsReview record);

    int updateByPrimaryKey(OsGoodsReview record);
}

File: src/main/kotlin/top/charjin/shoppingserver/mapper/OsGoodsReviewMapper.kt
package top.charjin.shoppingserver.mapper

import top.charjin.shoppingserver.entity.OsGoodsReview

interface OsGoodsReviewMapper {
    fun deleteByPrimaryKey(id: Int?): Int

    fun insert(record: OsGoodsReview): Int

    fun insertSelective(record: OsGoodsReview): Int

    fun selectByPrimaryKey(id: Int?): OsGoodsReview

    fun updateByPrimaryKeySelective(record: OsGoodsReview): Int

    fun updateByPrimaryKey(record: OsGoodsReview): Int
}